examples/with-validation/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/ls6-events/astra"
+	"github.com/ls6-events/astra/inputs"
+	"github.com/ls6-events/astra/outputs"
+)
+
+func main() {
+	r := gin.Default()
+
+	r.GET("/posts", GetPosts)
+	r.GET("/posts/:id", GetPost)
+	r.POST("/posts", CreatePost)
+	r.PUT("/posts/:id", UpdatePost)
+	r.DELETE("/posts/:id", DeletePost)
+
+	r.GET("/health", func(c *gin.Context) {
+		c.JSON(200, gin.H{
+			"status": "ok",
+		})
+	})
+
+	gen := astra.New(inputs.WithGinInput(r), outputs.WithOpenAPIOutput("openapi.generated.yaml"))
```

Comment:
Is it worth allowing the validation tags to be enabled/disabled as an option?

Reply:
Potentially, have we decide on a way we want to do this more generally? Do you think users would need disable this often?

Reply:
We're assuming that they're using govalidator as their validation method with their tags, so an override might be nice (I can see other libraries using `validate` tag with different methods)

Reply:
Yeah that's fair, I think in the future what would be nice is a way to specify all of these things on an input by input basis because gin will do things differently to all other frameworks, especially tag names and things like that.

Reply:
I think we should bench this for now as I want to get this PR in and I think it should come in with a whole other host of changes.